tests/integration/pilot/gateway_conformance_test.go

```diff
@@ -1,17 +1,3 @@
-// Copyright Istio Authors
```

Comment:
is this intentional?

Reply:
yes, it was there twice for no reason

Reply:
It would fail the linter if it was unexpected